assemblies/assembly-configuring-postgresql-databases.adoc

```diff
@@ -0,0 +1,21 @@
+[id='assembly-configuring-postgresql-databases_{context}']
+= Configuring external PostgreSQL databases
+
+As an administrator, you can configure and use external PostgreSQL databases in {product}. You must have a PostgreSQL certificate file that is used to configure the {product-very-short} instance either using the Operator or Helm Chart.
```

Comment:
You are talking about "TLS/SSL Certificate for PostgreSQL server" I think.
Well, strictly speaking you must not have it, we just recommend it for prod use and give an instruction how to configure it, but, technically, it can work w/o it.